openedx_events/event_bus/avro/tests/schemas/org+openedx+content_authoring+library_block+created+v1_schema.avsc

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "CloudEvent",
+  "type": "record",
+  "doc": "Avro Event Format for CloudEvents created with openedx_events/schema",
+  "fields": [
+    {
+      "name": "library_block",
+      "type": {
+        "name": "LibraryBlockData",
+        "type": "record",
+        "fields": []
```

Comment:
This doesn't seems right. For the `XBlockData`, the `fields` is not empty.

Reply:
Yep -- you'll need to add the `name` and `type` for each of your data fields in there.